plugins/dell-k2/fu-dell-k2-ec.c

```diff
@@ -850,6 +926,7 @@ fu_dell_k2_ec_init(FuDellK2Ec *self)
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_USABLE_DURING_UPDATE);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_DUAL_IMAGE);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SELF_RECOVERY);
+	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_INSTALL_SKIP_VERSION_CHECK);
```

Comment:
so the intention is not to do any end-to-end testing?

Reply:
Yeah, for now the dock reboot command for online activation yet ready. I plan to add it later when it becomes available.